protos/telemetry/telemetry.proto

```diff
@@ -270,3 +277,12 @@ message ActuatorOutputStatus {
     uint32 active = 1;
     repeated float actuator = 2;
 }
+
+// Landed State enumeration
+enum LandedState{
+    LANDED_UNKNOWN = 0;
+    LANDED_ON_GROUND = 1;
+    LANDED_IN_AIR = 2;
```

Comment:
`LANDED_IN_AIR` sounds weird to me. Could we instead use `LANDED_STATE` as a prefix? `LANDED_STATE_UNKNOWN`, `LANDED_STATE_IN_AIR`, ...

Reply:
Sure, LANDED_STATE prefix is perfectly fine. I have also made some changes to MAVSDK and mavsdk python to enable the landed_state enum, hopefully I can contribute accordingly after the PR is accepted. Thanks